package.json

```diff
@@ -1,14 +1,17 @@
 {
   "scripts": {
     "test": "sass-lint static/**/*.scss --verbose --no-exit",
-    "build": "node-sass --include-path node_modules static/sass --output static/css",
-    "watch": "node-sass --include-path node_modules --source-map true --watch static/sass --output static/css",
+    "build": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace static/css/*.css",
+    "watch": "watch -p 'static/sass/*.scss' -c 'yarn run build'",
     "clean": "rm -rf node_modules yarn-error.log static/css *.log *.sqlite _site/ build/ .jekyll-metadata"
   },
   "author": "Canonical webteam",
   "devDependencies": {
+    "autoprefixer": "^7.1.3",
+    "postcss-cli": "^4.1.0",
     "node-sass": "^4.5.3",
     "sass-lint": "^1.10.2",
-    "vanilla-framework": "1.5.1"
+    "vanilla-framework": "1.5.1",
+    "watch-cli": "^0.2.2"
   }
-}
\ No newline at end of file
+}
```

Comment:
I think we keep having diffs like this.
I suppose my editor is adding new lines at the end of file by default, while yours is removing them ;)

Reply:
I don't think my editor does anything with them as I'm usually using vim, but `package.json` often gets generated/updated by yeoman - with `yo canonical-webteam:run`. I'm guessing that removes newlines.

Reply:
OK. Maybe my editor (Atom) adds them to edited files. As I've seen added newlines in some files that I touched.